bundles/EcommerceFrameworkBundle/OrderManager/OrderManager.php

```diff
@@ -709,15 +703,17 @@ protected function buildTaxArray(array $taxItems)
      * Build order item key from cart item
      *
      * @param ICartItem $item
+     * @param bool $isGiftItem
      *
      * @return string
      */
-    protected function buildOrderItemKey(ICartItem $item)
+    protected function buildOrderItemKey(ICartItem $item, bool $isGiftitem = false)
```

Comment:
I'm not sure I agree since the change includes a default value for the second argument. But I can accept your verdict

What's your suggestion then? Should we modify the $item class or object instead to include the $isGiftItem or change the key outside of the method like in 6c31611?

Reply:
we should add a upgrade note and wait for that at least until 5.7